src/openrct2/windows/ride.c

```diff
@@ -6009,11 +6009,14 @@ static void window_ride_income_invalidate(rct_window *w)
 	w->pressed_widgets &= ~(1 << WIDX_PRIMARY_PRICE_SAME_THROUGHOUT_PARK);
 	w->disabled_widgets &= ~(1 << WIDX_PRIMARY_PRICE);
 
+	window_ride_income_widgets[WIDX_PRIMARY_PRICE].tooltip = STR_NONE;
+
 	//If the park doesn't have free entry, lock the admission price, unless the cheat to unlock all prices is activated.
 	if ((!(gParkFlags & PARK_FLAGS_PARK_FREE_ENTRY) && rideEntry->shop_item == SHOP_ITEM_NONE && ride->type != RIDE_TYPE_TOILETS)
 		&& (!gCheatsUnlockAllPrices))
 	{
 		w->disabled_widgets |= (1 << WIDX_PRIMARY_PRICE);
+		window_ride_income_widgets[WIDX_PRIMARY_PRICE].tooltip = STR_RIDE_INCOME_ADMISSION_PAY_FOR_ENTRY_TIP;
 	}
 
```

Comment:
I don't quite understand what you mean with this suggestion. Do you mean that it might be helpful if a tooltip is always shown? Even when the price spinner is disabled?